server/utils/createApplicationErrorClass.js

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
thanks for writing this class and and moving towards better error handling/capturing ✨